Gemfile.lock

```diff
@@ -185,6 +185,8 @@ GEM
     faraday (0.12.2)
       multipart-post (>= 1.2, < 3)
     ffi (1.10.0)
+    font-awesome-sass (5.8.1)
+      sassc (>= 1.11)
```

Comment:
Re-reviewing this pull request I've noticed we include the `sassc` gem. We had some problems when adding the `sassc` to Madrid's preproduction server and had to undo those changes :thinking:. What do you think about waiting for pull request #3286 to be tested and merged before adding this one?

Reply:
I've rebased this pull request now that we're using `sassc` :smile: